platform/src/pages/resources/controller.rs

```diff
@@ -0,0 +1,10 @@
+#[derive(Debug, Clone, PartialEq, Copy)]
+pub struct Controller {}
+
+impl Controller {
+    pub fn init(_lang: dioxus_translate::Language) -> Self {
```

Comment:
```suggestion
    pub fn new(_lang: dioxus_translate::Language) -> Result<Self, RenderError> {
```
로 앞으로는 controller 를 이렇게 정의해주세요.

Reply:
넵 앞으로 그렇게하도록 하겠습니다!